### ISSUE TITLE
React-jss with Next.js style issues

### ISSUE THREAD

**Issue by DMaxence** [hxf31891/react-gradient-color-picker#84]
I'm using Next.js 14 and I think because of `react-jss` the styles are not loaded even if I use it in client mode with `"use client"`.

I have in my console this warning :

<img width="441" alt="Capture d’écran 2024-02-07 à 10 53 00" src="https://github.com/hxf31891/react-gradient-color-picker/assets/23030942/e4b56687-99e9-4ede-8082-5127d471de2c">

 
The classes seems generated but there's no style in it.
<img width="841" alt="Capture d’écran 2024-02-07 à 10 54 21" src="https://github.com/hxf31891/react-gradient-color-picker/assets/23030942/0d8c3bd6-17e7-49bc-a2e1-4c9d33e46921">

I created a sandbox with the bug so you can see https://codesandbox.io/p/devbox/empty-snowflake-td4xnv

**Comment by ewan-kerboas** [comment]
Same problem here

**Comment by 7sne** [comment]
Same here

**Comment by hxf31891** [comment]
Apologies all, CSS & Next has been a persistent issue with this project. I thought JSS would work but apparently not. 
Regardless will try and get this fixed ASAP.

**Comment by nicosh** [comment]
> Apologies all, CSS & Next has been a persistent issue with this project. I thought JSS would work but apparently not. Regardless will try and get this fixed ASAP.

My i ask you why you choose to use JSS instead of  CSS Modules ?

**Comment by hxf31891** [comment]
@nicosh Next refuses to import style sheets used within npm packages

**Comment by nicosh** [comment]
@hxf31891 What you mean?  You [can import external stylesheets](https://nextjs.org/docs/app/building-your-application/styling/css-modules) without any problem in next.js (as many libraries does).
However my question  was about CSS modules that are supported natively by react and next without the need of external packages. 
I've forked the repo and everything work fine by just changing one line of code : 
```
  const useStyles = createUseStyles(styles)
  const classes = useStyles()
```
with 

`import styles from '../main.module.css'`

**Comment by hxf31891** [comment]
@nicosh Could you create a PR with your changes please?

**Comment by nicosh** [comment]
@hxf31891 https://github.com/hxf31891/react-gradient-color-picker/pull/88

I have not tested the final build

**Comment by hxf31891** [comment]
@nicosh thanks I will test it out

**Comment by hxf31891** [comment]
Alright I believe this to be fixed in the latest version v3.0.6

Big thanks @nicosh, much appreciated.

I will leave this open for a bit please let me know if the issue persists or others arrive.

One thing to note is that at least for now I removed disableDarkMode which was over complicating things. Will try and bring back in a later release if anybody is using it.

**Comment by hxf31891** [comment]
@nicosh this should be fixed as of 3.0.10
